### DISCUSSION TITLE
database backups

### DISCUSSION THREAD

**Discussion by iamfoysal** [cookiecutter/cookiecutter-django#5462] [Q&A]
How to backup database using docker outside of the project directory?

**Comment by xjlin0** [comment]
Please change production.yml to whatever you are using. Here is how I dump db to sql files.
* backup current db to container
```
docker compose -f production.yml exec postgres backup
```
* list backup files in container
```
docker compose -f production.yml exec postgres backups
```
* When postgres container is up, copy all backup files from container to host (backups folder)
```
docker cp $(docker compose -f production.yml ps -q postgres):/backups ./backups
```
